lib/components/result-view/result-view.js

```diff
@@ -28,12 +28,31 @@ class ResultViewComponent extends React.Component {
   containerTooltip = new CompositeDisposable();
   buttonTooltip = new CompositeDisposable();
   expanded: IObservableValue<boolean> = observable(false);
+  ctrlHovering: IObservableValue<boolean> = observable(false);
 
   getAllText = () => {
     if (!this.el) return "";
     return this.el.innerText ? this.el.innerText.trim() : "";
   };
 
+  handleClick = (event: MouseEvent) => {
+    if (this.ctrlHovering.get()) {
```

Comment:
Yep, that's how my alt PR will work.

The reason for the observable was to keep the icon class in sync with whatever result would occur when clicking ( I couldnt get the icon to respond to ctrl keydown alone). It is hard to explain, and maybe not worth the trouble :laughing: